pkg/util/logging/default_logger.go

```diff
@@ -19,9 +19,12 @@ package logging
 import (
 	"os"
 
+	"github.com/heptio/velero/pkg/cmd/util/flag"
 	"github.com/sirupsen/logrus"
 )
 
+const defaultLogFormat = "text"
```

Comment:
let's define an enum here, like this:

```go
type Format string

const (
    FormatText Format = "text"
    FormatJSON Format = "json"
)
```

And then instead of passing strings around, we'll pass around a var of type `logging.Format`